hdfs3/core.py

```diff
@@ -271,8 +271,8 @@ def info(self, path):
         _lib.hdfsFreeFileInfo(ctypes.byref(fi), 1)
         return out
 
-    def glob(self, path):
-        """ Get list of paths mathing globstring (i.e., with "*"s).
+    def walk(self, path):
+        """ Get list of paths mathing glob-like pattern (i.e., with "*"s).
```

Comment:
I think that both glob and walk are independently useful.  Glob uses `*`s, walk assumes that you select a directory.